tensorflow/core/kernels/data/experimental/map_and_batch_dataset_op.cc

```diff
@@ -125,6 +125,8 @@ class MapAndBatchDatasetOp::Dataset : public DatasetBase {
   }
 
   int64 Cardinality() const override {
+    if (!preserve_cardinality_)
```

Comment:
Even though the `if` contains only a single line, please use curly braces to be consistent with the rest of the file.

Reply:
fixed.